tests/core/config/inline.js

```diff
@@ -22,6 +22,10 @@ bender.test( {
 		} );
 	},
 
+	test_startup_computed_state_value: function() {
+		assert.isTrue( this.editor.config.useComputedState === true, 'config.useComputedState should return true as default' );
```

Comment:
There is no need to compare variable's value to `true`, asserting its value is enough in this case:

```suggestion
		assert.isTrue( this.editor.config.useComputedState, 'config.useComputedState should return true as default' );
```

Reply:
Done